lib/datasource/conan/index.ts

```diff
@@ -15,8 +18,35 @@ export class ConanDatasource extends Datasource {
 
   override readonly registryStrategy = 'merge';
 
-  constructor() {
-    super(ConanDatasource.id);
+  githubHttp: GithubHttp;
+
+  constructor(id = ConanDatasource.id) {
+    super(id);
+    this.githubHttp = new GithubHttp(id);
+  }
+
+  async getConanCenterReleases(
+    depName: string,
+    userAndChannel: string
+  ): Promise<ReleaseResult | null> {
+    if (userAndChannel && userAndChannel !== '@_/_') {
+      logger.debug(
+        { depName, userAndChannel },
+        'User/channel not supported for Conan Center lookups'
+      );
+      return null;
+    }
+    const url = `https://api.github.com/repos/conan-io/conan-center-index/contents/recipes/${depName}/config.yml`;
+    const res = await this.githubHttp.getJson<{ content: string }>(url);
```

Comment:
Add `{headers: { Accept: 'application/vnd.github.v3.raw' }}` and you can skip the base64 decoding step below.

Source: https://docs.github.com/en/rest/reference/repos#custom-media-types-for-repository-contents

Reply:
I didn't see you had set automerge. I'll resolve this conversation to get this merged, and send out a follow-up PR.